src/scss/templates/_blog.scss

```diff
@@ -112,6 +112,24 @@
 			background-color: white !important;
 		}
 	}
+
+	.blog-post-table {
```

Comment:
too much nesting please max 3, but I think you can do that with 2 too

Reply:
> too much nesting please max 3, but I think you can do that with 2 too

Ok, let me modify it again.